aioasuswrt/asuswrt.py

```diff
@@ -247,6 +248,8 @@ def __init__(
         self._devices_cache = None
         self._transfer_rates_cache = None
         self._latest_transfer_data = 0, 0
+        self._nvram_cache_timer = None
```

Comment:
My preference would be to name this `_nvram_cache_time` seeing that it's an actual time, and not really a timer.

Reply:
As comment before,  used the same naming used for similar variable. I can change as suggested, but make sense have different convention inside the module?

Reply:
Lets go with what we have!
If we want to change it we can go for a seperate PR